dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderImpl.java

```diff
@@ -515,12 +530,12 @@ private boolean hasWrongSubjectRDNs(List<X509Certificate> certificateChain) thro
 	 */
 	private boolean isExpired(List<X509Certificate> certificateChain) {
 		X509Certificate certificate = getLeafCertificate(certificateChain);
-		if (certificateChain == null) {
+		if (certificate == null) {
```

Comment:
Good catch here. But I am wondering if we shouldn't check the whole certificate chain? What happens when the intermediate or root ca cert expires before our leaf cert?

Reply:
(FYI - we can consider that in another issue, since we probably have other areas we need to consider this in)

Reply:
I opened an issue.